importer/src/main/java/org/hiero/mirror/importer/config/ImporterConfiguration.java

```diff
@@ -45,7 +51,17 @@ LeaderService leaderService() {
     }
 
     @Bean
-    FlywayConfigurationCustomizer flywayConfigurationCustomizer() {
+    @ConfigurationProperties(prefix = "spring.flyway.hikari")
+    @Owner
+    HikariConfig flywayHikariConfig(JdbcConnectionDetails connectionDetails) {
+        var config = new HikariConfig();
+        config.setJdbcUrl(connectionDetails.getJdbcUrl());
+        return config;
+    }
```

Comment:
Do we really need to be able to configure all the properties separately at `spring.flyway.hikari` and have to duplicate them in the yml? Instead, can't we inject the existing HikariConfig and duplicate it?

```suggestion
    @Owner
    HikariDataSource flywayDataSource(JdbcConnectionDetails connectionDetails, HikariConfig hikariConfig) {
        var flywayConfig = new HikariConfig();
        hikariConfig.copyStateTo(flywayConfig);
        flywayConfig.setJdbcUrl(connectionDetails.getJdbcUrl());
        flywayConfig.setPoolName("flyway");
        return flywayConfig;
    }
```